pkg/plugin/manager.go

```diff
@@ -65,8 +66,8 @@ func (m *Manager) DownloadProviders(ctx context.Context, providers []registry.Pr
 	log.Debug().Interface("providers", providers).Msg("Downloading required providers")
 	downloaded := make([]registry.ProviderBinary, len(providers))
 	for i, rp := range providers {
-		if _, ok := m.clients[rp.String()]; ok {
-			log.Debug().Str("name", rp.Name).Str("version", rp.Version).Msg("Skipping provider download, using reattach instead")
+		if _, ok := m.clients[rp.Name]; ok {
+			ui.ColorizedOutput(ui.ColorInfo, fmt.Sprintf("Skipping provider %s download, using reattach instead\n", ui.Colorize(ui.ColorSuccessBold, false, rp.Name)))
```

Comment:
I think this is quite a useful log message to have to ensure the re-attach was successful